src/scripts/content.js

```diff
@@ -18,17 +18,57 @@ const createTeamsNotificationScript = () => {
   (document.head || document.documentElement).appendChild(s);
 };
 
-chrome.storage.sync.get(defaultOptions, optionsFromStorageOrDefaults => {
-  teamsExtension.options = optionsFromStorageOrDefaults;
-  console.debug('Teams extension options', teamsExtension.options);
-
-  createTeamsNotificationScript();
-});
+const createFocusLostHandler = windowId => () => {
+  console.debug('Focus lost: ' + windowId);
+  if (teamsExtension.options.switchToNewChatOnBlur) {
+    // This timeout is necessary to be able to focus a Teams window by clicking its notification. Otherwise it just minimizes right away.
+    setTimeout(() => {
+      if (!document.hasFocus()) {
+        // Don't switch to New Chat if there is a call is running
+        const hangupButton = top.document.querySelector('iframe.embedded-page-content').contentDocument.getElementById('hangup-button');
+        if (!hangupButton) {
+          const buttonNewChat = top.document.querySelector('[title="Nuova chat (Alt+N)"]');
+          if(buttonNewChat){
+            const elemNewChat = document.querySelector('[data-tid="chat-li-entry-with-Nuova chat"]');
```

Comment:
Another language-dependent selector. I would try to select the <a> tag within this div, using the selector on the class `cle-new`, which seems to be used only for new chat.